tests/test_helpers/test_search/test_models.py

```diff
@@ -167,67 +169,164 @@ def test_description():
 
 def test_constraint_type():
     """Test ConstraintType."""
-    for cons_type in [
-        ConstraintTypes.EQUAL,
-        ConstraintTypes.NOT_EQUAL,
+    constraint_type_values = {
+        "int": 12,
+        "bool": True,
+        "float": 10.4,
+        "str": "some_string",
+        "location": Location(1.1, 2.2),
```

Comment:
Please have some tests which explicitly use the keywords `latitude` and `longitude` and assert encoding/decoding leads to correct recovery!

Reply:
Done